core/config.js

```diff
@@ -356,6 +356,17 @@ CKEDITOR.config = {
 	 */
 	protectedSource: [],
 
+	/**
+	 * If set to true, **disables** CKEditor.protectedSource default restrictions
+	 * for tags like script, noscript and other. **This can potentially allow execution
+	 * of malicious scripts inside the editor**. Use with caution.
+	 *
+	 *		config.allowDangerousProtectedSoruce = true;
+	 *
+	 * @cfg
+	 */
+	allowDangerousProtectedSource: false,
```

Comment:
WDYT about renaming it to something like `allowDangerousTags`? Since it has effect only on hard coded protected tags and doesn't have any influence on how `editor.config.protectedSource` it is somehow a separate concept.

And then I will modify docs to something like:

```
	/**
	 * If set to true, **disables** default protected tags restrictions allowing `script`, `noscript` and `meta` tags. 
     * **This can potentially allow execution of malicious scripts inside the editor**. Use with caution.
	 *
	 *		config.allowDangerousTags = true;
	 *
	 * @cfg
	 */
```

Reply:
Updated.